website/docs/search.md

```diff
@@ -182,6 +187,30 @@ If you prefer to edit the Algolia search React component, swizzle the `SearchBar
 npm run swizzle @docusaurus/theme-search-algolia SearchBar
 ```
 
+## Using Typesense DocSearch {#using-typesense-docsearch}
```

Comment:
It would be helpful for users to provide a link to a live Docusaurus site using Typesense in this section, it would help figure out if the end result is good enough before investing any time?

(I'd suggest forking Docusaurus site, replacing the algolia plugin by typesense, and deploying to Netlify + provide live URL and source)

My intuition is that the UI/UX would be similar to Algolia, but can't be sure. For users, it's harder to assess Typesense than Algolia